AirJoyData:
  topic_name: /ibus_data
  queue_size: 1000